.github/release.yml

```diff
@@ -0,0 +1,15 @@
+changelog:
+  categories:
+    - title: Breaking Changes
+      labels:
+        - testnet-restart
+    - title: Updates
```

Comment:
Maybe, _Other Changes_?

Reply:
What if there a no protocol breaking changes? This is the default category, which in the future should be the main category once there are no more breaking changes.

Reply:
Yeah, not sure. To me _Updates_ sounds more like dependency updates but this is not that

Reply:
@sisou Will change it to "What's changed" to merge this PR. wdyt?

Reply:
Sure!